lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -216,8 +216,7 @@ module.exports = [
       },
       audits: {
         'http-status-code': {
-          score: 0,
-          displayValue: '403',
+          score: null,
```

Comment:
seems like this audit is useless at this point now, candidate for deletion?

Reply:
> seems like this audit is useless at this point now, candidate for deletion?

I'd say yes (are there cases when non 200 status codes shouldn't be a runtimeError?)